src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -1022,20 +1021,20 @@ private Properties setCommonConnectionParameters(boolean validateDefaultParamete
     return kvMap2Properties(params, validateDefaultParameters);
   }
 
-  @Test
-  public void testFailOverOrgAccount() throws SQLException {
-    // only when set_git_info.sh picks up a SOURCE_PARAMETER_FILE
-    assumeTrue(RunningOnGithubAction.isRunningOnGithubAction());
-
-    Map<String, String> kvParams = getConnectionParameters(null, "ORG");
-    Properties connProps = kvMap2Properties(kvParams, false);
-    String uri = kvParams.get("uri");
-
-    try (Connection con = DriverManager.getConnection(uri, connProps);
-        Statement statement = con.createStatement()) {
-      statement.execute("select 1");
-    }
-  }
+  //  @Test
```

Comment:
It could be better to use @Ignore than comment the code.

Reply:
Done.